### PR TITLE
add sept coworking

### PR DESCRIPTION
**Pull request by steffilazerte**

### CODE REVIEW

File: content/events/2023-09-05-coworking-2023-09.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Social Coworking and Office Hours - AI in/for R
+dateStart: 2023-09-05T01:00:00 # UTC!!
+dateEnd: 2023-09-05T03:00:00 # UTC!!
+date: 2023-09-05T03:00:00 # UTC!! same as dateEnd
+description: Monthly coworking for productivity, asking questions, socializing
```
Comment (maelle) [inline content/events/2023-09-05-coworking-2023-09.md:6]: @steffilazerte maybe add a sentence more specific to the topic? See https://deploy-preview-613--ropensci.netlify.app/coworking/

Comment (steffilazerte) [reply content/events/2023-09-05-coworking-2023-09.md:6]: Ah right! I forgot about that change! Thanks for the reminder

Comment (maelle) [reply content/events/2023-09-05-coworking-2023-09.md:6]: I rebased your PR to check the new coworking page wasn't broken :grin:

Comment (steffilazerte) [reply content/events/2023-09-05-coworking-2023-09.md:6]: Thank you!

**Pull request merged**